test/perf-test/perf.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
```

Comment:
would it make sense to have a "setup" function here ? Right now this script assumes that the user will have to have configured the target cluster already.

Reply:
The script assumes `oc login` has occurred already, and the `codeflare-stack` kfdef has been applied (which itself assumes that the ODH operator and codeflare operators have been installed.)

I can have my script query someone for their oc login token/server, login for them, and then maybe apply the required operators and kfdefs prior to running the performance script.   Is that what you're thinking?

Reply:
Not exactly the same but close. I was thinking that the script validates that there's a kube/config in the user's home directory, and then validate that the CRD(s) and controller is running (we might want to have the namespace where the controller is deployed).  We should probably support a `-h` command line argument that would explain to the user what are the preconditions for running this script.

Reply:
OK - I check that the login is completed, otherwise it asks you to login.

Shoot, I forgot to check that the MCAD controller is installed/running and has the CRDs - let me see if I can add that.

Reply:
Done